src/tests.rs

```diff
@@ -15,6 +15,23 @@ fn test_load() {
 }
 
 #[test]
+fn test_load_debug() {
+    let lua = Lua::new();
+    lua.eval::<()>("debug", None).unwrap();
```

Comment:
This doesn't do anything regardless of the value of `debug`. Try to just `exec("assert(debug == nil)").unwrap()`

Reply:
Nice catch, fixed